service/create/templates.go

```diff
@@ -137,7 +137,7 @@ USERDATA_FILE={{.MachineType}}
     --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
     --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
     --trust-keys-from-https \
-    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3DirURI}}/$USERDATA_FILE /var/run/coreos/temp.txt
+    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3URI}}/$USERDATA_FILE /var/run/coreos/temp.txt
```

Comment:
What about backwards compatibilities? Are there raw templates out there that might break in between? Remember that we have to start thinking about such things. Not the most critical thing right now. Just saying.